scripts/poststart_hook.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# Configuration
+PLATFORM="docker"  # Change to "helm" or "cdk" as needed
+VERSION="unknown"
+STATUS=""
+SERVER_HEALTH_URL="http://hyperswitch-server:8080/health"
+HYPERSWITCH_URL="http://hyperswitch-server:8080/health/ready"
+WEBHOOK_URL="https://hyperswitch.gateway.scarf.sh/$PLATFORM"
+
+# Fetch health status
+echo "Fetching app server health status..."
+HEALTH_RESPONSE=$(curl -s --fail "$SERVER_HEALTH_URL") || HEALTH_RESPONSE="connection_error"
+
+if [ "$HEALTH_RESPONSE" = "connection_error" ]; then
+    STATUS="error"
+    ERROR_MESSAGE=$(echo "404 response" | jq -sRr @uri)
+    PARAMS="$PARAMS&status=$STATUS"
+    PARAMS="$PARAMS&error_message=$ERROR_MESSAGE"
+    curl -G "$WEBHOOK_URL?$PARAMS"
+    exit 0
+fi
+
+#fetch hyperswitch version
+VERSION=$(curl --silent --output /dev/null --request GET --write-out '%header{x-hyperswitch-version}' "$HYPERSWITCH_URL" | sed 's/-.*//' | jq -sRr @uri)
```

Comment:
1. The `sed` command here would drop the hotfix part in tags like `2025.04.02.0-hotfix1`.
   Can we only drop the `-dirty` part specifically?
2. Expand the short `jq` flags.